dabudimesh/shell.py

```diff
@@ -1,12 +1,18 @@
 from utils import create_connection
+from message import Message
 import bluetooth
 from cmd import Cmd
 
 
 def _on_read(sock, router):
-    text = router.process_message_from(sock)
-    if text is not None:
-        print(text)
+    message = router.process_message_from(sock)
```

Comment:
I think it's better to handle messege type in router and return message if it has text that may be used by user